lib/Conversion/TorchToTcp/Elementwise.cpp

```diff
@@ -304,6 +304,48 @@ class ConvertAtenSqrtOp : public OpConversionPattern<AtenSqrtOp> {
   }
 };
 
+class ConvertAtenCeilOp : public OpConversionPattern<AtenCeilOp> {
```

Comment:
These conversions are very similar to `tcp::SqrtOp` and `tcp::TanhOp`. We should refactor all of these into a single function that is templatized on the aten and tcp ops. So, we don't have to duplicate the code.

Reply:
I can create a followup PR to make a templated function for this.